.golangci.yml

```diff
@@ -6,8 +6,9 @@ run:
 
 linters:
   enable:
-    - gofumpt
     - errorlint
+    - nolintlint
+    - gofumpt
```

Comment:
The commit msg says the errorlint finally can ignore those errors. But how is that happening? Did we change the version somewhere I'm not seeing?

Reply:
errorlint is bundled in golangci-lint, which we update from time to time. Any changes in either errorlint or golangci-lint can result it those being less strict.

Now, this PR adds nolintlint which reports about annotations that are no longer required.

As for the removed annotations, they are mostly result of https://github.com/polyfloyd/go-errorlint/pull/47 and https://golangci-lint.run/usage/false-positives/#exc0001.

Reply:
Ohhh, now I understand. Thanks!